Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//TIP To <b>Run</b> code, press <shortcut actionId=\"Run\"/> or\r\n// click the <icon src=\"AllIcons.Actions.Execute\"/> icon in the gutter.\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        User.addUser();\r\n        System.out.println(User.userList.get(0).getUserName());\r\n        User.userList.get(0).removeUser();\r\n        System.out.println(User.userList.get(0).getUserName());\r\n\r\n        User u1 = new User(\"2\", \"Josefina\", \"Administrator\");\r\n        System.out.println(u1.getUserName());\r\n        u1.removeUser();\r\n        System.out.println(u1.getUserName());\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision c08236ebd93a6a8db3b1693417a6201accbce426)
+++ b/src/Main.java	(date 1743084122792)
@@ -2,15 +2,6 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        User.addUser();
-        System.out.println(User.userList.get(0).getUserName());
-        User.userList.get(0).removeUser();
-        System.out.println(User.userList.get(0).getUserName());
-
-        User u1 = new User("2", "Josefina", "Administrator");
-        System.out.println(u1.getUserName());
-        u1.removeUser();
-        System.out.println(u1.getUserName());
-
+       
     }
 }
\ No newline at end of file
